app/models/host/base.rb

```diff
@@ -370,5 +371,23 @@ def check_provision_interface
         errors.add :interfaces, _("managed host must have one provision interface")
       end
     end
+
+    # we can't use standard unique validation on interface since we can't properly handle :scope => :host_id
+    # for new hosts host_id does not exist at that moment, validation would work only for persisted records
+    def uniq_interfaces_identifiers
+      success = true
+      identifiers = []
```

Comment:
host does not have `identifiers` method AFAIK, interfaces have one similar, called `identifier`... anyway I can't think of better name for variable that holds interface identifiers